contentcuration/contentcuration/static/js/edit_channel/utils/workspace_manager.js

```diff
@@ -0,0 +1,70 @@
+var _ = require("underscore");
+var Models = require("edit_channel/models");
+
+function WorkspaceManager(){
```

Comment:
My solution for getting relationships between DOM elements and views without relying on jquery too much